app/controllers/concerns/raw_display.rb

```diff
@@ -11,8 +11,11 @@ def render_display(blob, url_options: {})
       renderer = Seek::Renderers::RendererFactory.instance.renderer(blob, url_options: url_options)
     end
     if renderer.can_render?
-      response.set_header('Content-Security-Policy', renderer.content_security_policy)
-      render renderer.format => renderer.render_standalone.html_safe, layout: renderer.layout
+      # check if allowed by cookies
+      unless renderer.is_remote? && !cookie_consent.allow_embedding?
+        response.set_header('Content-Security-Policy', renderer.content_security_policy)
+        render renderer.format => renderer.render_standalone.html_safe, layout: renderer.layout
+      end
```

Comment:
What does this actually render if embedding is forbidden?

Would be nice if it displayed some kind of page with a message.